docs/experiments-msmarco-doc.md

```diff
@@ -144,3 +144,4 @@ We can see that Anserini's (tuned) BM25 baseline is already much better than the
 + Results reproduced by [@mayankanand007](https://github.com/mayankanand007) on 2021-05-04 (commit [`a9d6f66`](https://github.com/castorini/pyserini/commit/a9d6f66234b5dd2859a0dc116ef3e38a52d0f81d))
 + Results reproduced by [@rootofallevii](https://github.com/rootofallevii) on 2021-05-14 (commit [`e764797`](https://github.com/castorini/pyserini/commit/e764797081eebf487fa7e1fa34872a59ff97fdf7))
 + Results reproduced by [@jpark621](https://github.com/jpark621) on 2021-06-13 (commit [`f614111`](https://github.com/castorini/pyserini/commit/f614111f014b7490f75e585e610f64f769164dd2))
++ Results reproduced by [@mzzchy](https://github.com/mzzchy) on 2021-07-05 (commit [`dc730ff`](https://github.com/castorini/pyserini/commit/dc730ffb24e1ff2a34999166ac4d14e41b8312d5))
```

Comment:
seems this commit id does not belong to this repo? 
>This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Reply:
That URL actually redirects me to my commit to this pr.
Here is how I create the pr.

Add a record to md file with a random commit id.
Change the commit id to the one generated in step 1.
Fix up/Squash the commit.
I don't think there is a way to get the commit id before I make a commit. Or is there a better way to do this

Reply:
the commit id here should be the last commit of the master branch in this repo, from where you did the replication.